src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml

```diff
@@ -1,74 +1,98 @@
+                        <RowDefinition />
+                        <RowDefinition />
+                    </Grid.RowDefinitions>
+                    <Label
+                        Grid.Row="0"
+                        Grid.Column="0"
+                        Text="Include barriers"
+                        VerticalTextAlignment="Center" />
+                    <Switch
+                        x:Name="IncludeBarriers"
+                        Grid.Row="0"
+                        Grid.Column="1"
+                        IsToggled="True"
+                        Toggled="IncludeBarriersChanged" />
+                    <Label
+                        Grid.Row="1"
+                        Grid.Column="0"
+                        Text="Include containers"
+                        VerticalTextAlignment="Center" />
+                    <Switch
+                        x:Name="IncludeContainers"
+                        Grid.Row="1"
+                        Grid.Column="1"
+                        IsToggled="True"
+                        Toggled="IncludeContainersChanged" />
+                </Grid>
+                <Label FontSize="Title" Text="Example barrier condition for this data: 'Transformer Load' Equal '15'" />
+                <Label Text="Define new condition" />
+                <Grid ColumnSpacing="10">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <Picker
+                        x:Name="Attributes"
+                        Grid.Row="0"
+                        ItemDisplayBinding="{Binding Name}"
+                        SelectedIndexChanged="OnAttributeChanged"
+                        VerticalOptions="CenterAndExpand" />
+                    <Picker
+                        x:Name="Operators"
+                        Grid.Column="1"
+                        HorizontalOptions="CenterAndExpand" />
+                    <Picker
+                        x:Name="ValueSelection"
+                        Grid.Column="2"
+                        HorizontalOptions="CenterAndExpand"
+                        ItemDisplayBinding="{Binding Name}" />
+                    <Entry
+                        x:Name="ValueEntry"
+                        Grid.Column="2"
+                        BackgroundColor="White"
+                        HorizontalOptions="CenterAndExpand"
+                        HorizontalTextAlignment="Center"
+                        IsVisible="False"
+                        MinimumWidthRequest="50"
+                        TextColor="Black" />
+                </Grid>
+                <Button Clicked="OnAddCondition" Text="Add condition" />
+                <Label FontSize="Title" Text="Barrier conditions" />
```

Comment:
I think the `FontSize="Title"` is a little large when compared to the other font sizes in this sample.